source/ch_protocols.rst

```diff
@@ -545,6 +545,10 @@ The first four channels represent Aileron, Elevator, Throttle, and Rudder.  Addi
 * Channel 10 engages the return-to-home feature
 * Channel 11 enables Take Off/Landing
 * Channel 12 activates emergency stop
+* Channel 13 analog aux channel #1
+* Channel 14 analog aux channel #2
+
+**NOTE:  Channels 13 and 14 are only active when using** `NFE Silverware firmware <https://github.com/NotFastEnuf/NFE_Silverware/>`_ **(and some other Silverware forks). Two otherwise static bytes in the protocol overridden to add two 'analog' (non-binary) auxiliary channels. The "Analog Aux" Bayang protocol option must be enabled on the TX and the Silverware firmware must have the Analog Aux feature enabled. Mismatches betwen TX and RX settings will prevent binding.**
```

Comment:
Your firmware changes have been added to nfe upstream repository. Please update text and link.

Reply:
Maybe I’m misunderstanding your comment, but the Silverware fork linked is the main NFE repository link.

The Analog Aux feature is not in the “original” Silverware repository, which has been stale since around August 2018. Is that the point of confusion?

The NFE fork linked is the most current and active Silverware fork.

If that doesn’t clear this up, could you be more specific what should change?

Thanks!

Reply:
I'm very sorry, I thought the link pointed to your fork.